force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -39,19 +39,34 @@ public with sharing class ClassesObjectsMethods {
      * @return A list containing the length of the string, a substring of the string, and the string in lower case.
      */
     public static List<String> practiceStringMethods() {
-        // Initialize a String with 'Hello, Salesforce!'
-        
-        // Get the length of the String        
-        // Get a substring of the String of the first 5 characters
-        // Convert the String to lower case
-        
-        // Return the list with the string data
-        // In the first position add the string length
-        // In the second position add the substring
-        // In the third position add the string in lower case
+         // Initialize a String with 'Hello, Salesforce!'
 
-        return null; // Replace null with the variable you used to store the result
-    }
+         String myString = 'Hello, Salesforce!';
+        
+         // Get the length of the String
+         Integer stringLength = myString.length();
+ 
+         // Get a substring of the String of the first 5 characters
+         String substringOfMyString = myString.substring(0, 5);
+ 
+         // Convert the String to lower case
+         String lowerCaseOfMyString = myString.toLowerCase();
+         
+         // Return the list with the string data
+         List<String> myList = new List<String>();
+ 
+ 
+         // In the first position add the string length
+         myList.add(String.valueOf(stringLength));
+ 
+         // In the second position add the substring
+         myList.add(substringOfMyString);
+ 
+         // In the third position add the string in lower case
+         myList.add(lowerCaseOfMyString);
+ 
+         return myList; // Replace null with the variable you used to store the result
+     }
```

Comment:
One of the things I post about a lot is creating variables you don't really need. In this case I think you could have assigned the results of your String calls directly to the list. 

``` public static List<String> practiceStringMethods() {
        List<String> strings = new List<String>();
        String helloSalesforce = 'Hello, Salesforce!';
        strings.add(String.valueOf(helloSalesforce.length()));
        strings.add(helloSalesforce.substring(0, 5));
        strings.add(helloSalesforce.toLowerCase());
        return strings;
    }
    ```